app/src/main/java/nl/avans/drivioapp/view/EditAdvertisement.kt

```diff
@@ -116,7 +116,9 @@ class EditAdvertisement : Fragment(R.layout.fragment_edit_advertisement) {
                 val response = advertisementViewModel.putAdvertisementResponse.value
 
                 if (response?.code() == 200) {
```

Comment:
Misschien handig om deze response door te laten verwijzen naar een ander fragment (zelfde voor andere plekken waar de response is)

Reply:
Ik had deze gezet om terug te gaan naar de details maar dan krijg je een leeg scherm. Voor nu heb ik het zo gelaten. Ik kan hem wel laten navigeren naar de MyAdvertisements fragment

Reply:
Reverted omdat er anders falende ui testen komen